server.js

```diff
@@ -1,9 +1,20 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const config = {
+	mongoURI:
+		'mongodb+srv://akshitp:E7lFFxThDg8fe7Dd@cluster0.6uq2j9y.mongodb.net/?retryWrites=true&w=majority'
+}
 
 const app = express();
 
 app.get('/ping', (req, res) => {
-    res.json({'message': 'pong'})
+    mongoose.connect(config.mongoURI)
```

Comment:
Close the MongoDB connection after the response is sent. Keeping the connection open might lead to memory leaks. [medium]